pinot-plugins/pinot-stream-ingestion/pinot-kinesis/src/main/java/org/apache/pinot/plugin/stream/kinesis/KinesisConfig.java

```diff
@@ -67,7 +67,7 @@ public class KinesisConfig {
 
   // TODO: this is a starting point, until a better default is figured out
   public static final String DEFAULT_MAX_RECORDS = "20";
-  public static final String DEFAULT_SHARD_ITERATOR_TYPE = ShardIteratorType.LATEST.toString();
+  public static final String DEFAULT_INIT_SHARD_ITERATOR_TYPE = ShardIteratorType.TRIM_HORIZON.toString();
```

Comment:
What does `TRIM_HORIZON` stand for? Is this similar to `EARLIEST`?

Reply:
yes, that is the terminology for EARLIEST in Kinesis